ale/drivers/kaguya_driver.py

```diff
@@ -64,42 +84,64 @@ def _sensor_orientation(self):
             self._orientation = qua
         return self._orientation.tolist()
 
-
     @property
-    def focal2pixel_lines(self):
+    def focal2pixel_samples(self):
         """
         Calculated using pixel pitch and 1/pixel pitch
         """
-        pixel_size = spice.gdpool('INS{}_PIXEL_SIZE'.format(self.ikid), 0, 1)[0]
+        pixel_size = spice.gdpool('INS{}_PIXEL_SIZE'.format(self.tc_id), 0, 1)[0]
         return [0, 0, 1/pixel_size]
 
 
     @property
-    def focal2pixel_samples(self):
+    def focal2pixel_lines(self):
         """
         Calculated using pixel pitch and 1/pixel pitch
         """
-        pixel_size = spice.gdpool('INS{}_PIXEL_SIZE'.format(self.ikid), 0, 1)[0]
+        pixel_size = spice.gdpool('INS{}_PIXEL_SIZE'.format(self.tc_id), 0, 1)[0]
         return [0, 1/pixel_size, 0]
 
 
     @property
-    def _odtx(self):
+    def _odtkx(self):
         """
         Returns
         -------
         : list
           Optical distortion x coefficients
         """
-        return spice.gdpool('INS{}_DISTORTION_COEF_X'.format(self.ikid),0, 4).tolist()
+        return spice.gdpool('INS{}_DISTORTION_COEF_X'.format(self.tc_id),0, 4).tolist()
 
 
     @property
-    def _odty(self):
+    def _odtky(self):
         """
         Returns
         -------
         : list
           Optical distortion y coefficients
         """
-        return spice.gdpool('INS{}_DISTORTION_COEF_Y'.format(self.ikid), 0, 4).tolist()
+        return spice.gdpool('INS{}_DISTORTION_COEF_Y'.format(self.tc_id), 0, 4).tolist()
+
+    @property
+    def line_exposure_duration(self):
+        # this is dumb
+        if isinstance(self.label['LINE_EXPOSURE_DURATION'], list):
+            return self.label['LINE_EXPOSURE_DURATION'][0].value * 0.001  # Scale to seconds
+        else:
+            return self.label['LINE_EXPOSURE_DURATION'].value * 0.001  # Scale to seconds
```

Comment:
It can possibly be more robust, unfortunately, the labels are inconsistent in that sometimes this is a list and sometimes not. There is probably a bigger conversation here about enforcing some of our datatype/structural assumptions. Until then, I think this is good enough. If there is another case other than the two here, then we just have to deal with it when we run into it.

Reply:
What if, to make it easier for the next person who may run into this that isn't one of us, we have an error message warning about this inconsistency? Would that be helpful or would it be over the top?

Reply:
Probably over the top for now. We need a better debug system (as mentioned in #95), I could probably throw an error at the end here but it'll probably be undone later anyways once we found something better and as we are the main consumers we can assume a certain level of intimacy with the code base until this becomes more production ready.